start/src/main/java/org/apache/accumulo/start/Main.java

```diff
@@ -51,26 +51,23 @@ public static void main(final String[] args) throws Exception {
       var deprecatedConfClass = org.apache.accumulo.start.classloader.AccumuloClassLoader
           .getClassLoader().loadClass("org.apache.hadoop.conf.Configuration");
       confClass = deprecatedConfClass;
+      Object conf = null;
```

Comment:
null initializer is redundant (as is the one for `confClass` at line 48)

Reply:
Redundant, but not an error / issue. I assume the compiler does the right thing. I didn't write it, I just moved lines around.

Reply:
While you are in the neighborhood - its a chance to remove a checkstyle warning and a) means we don't need to clean it up later and b) makes finding other checkstyle issues because there  would be 2 fewer if you fix both now.